pythonscript/_godot_io.pxi

```diff
@@ -51,15 +51,19 @@ class GodotIOStream(RawIOBase):
             except BaseException:
                 sys.__stderr__.write("Error calling GodotIOStream callback:\n" + traceback.format_exc() + "\n")
     
+    @staticmethod
+    def callback_id(callback):
+        return callback.__module__ + "__" + callback.__name__
```

Comment:
`f"{callback.__module__}__{callback.__name__}"` ?

Reply:
besides this function could be declared as `cdef inline` given it has no use outside of this module (or even use a [module-level inline function](https://cython.readthedocs.io/en/latest/src/userguide/pyrex_differences.html#cdef-inline))

Reply:
Aaaand finally what's the need to use this over `id` ? (I guess we should add a comment about this given it's not obvious)

Reply:
yep, will make module level inline.
I found that in my code the ids were changing (I have no idea why). Maybe some weird garbage-collection, or some duplicate instancing/multiple imports or something of the sort. anyway, id() was not constant and as a result the callback never got de-registered.